app/views/stories/create.js.erb

```diff
@@ -0,0 +1,7 @@
+<% if @story.errors.any? %>
+  $("#new_story_form").html("<%= j render('form', story: @story) %>");
+  rating_binder();
+<% else %>
+  $("#stories_table").prepend("<%= j render('shared/story_row', story: @story) %>");
+  $(".close-modal").click()
```

Comment:
you should add semi-colon at the end of this line

Reply:
Yes, Fixed